### PR TITLE
[Trading][Exchange] Add trading exchanges

### PR DESCRIPTION
**Pull request by Herklos**
Depends on https://github.com/Drakkar-Software/OctoBot-Trading/pull/103

Depends on https://github.com/Drakkar-Software/OctoBot-Trading/pull/129

### CODE REVIEW

File: Trading/Exchange/Margin/binance.pxd [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from octobot_trading.exchanges.margin.margin_exchange cimport MarginExchange
+
+cdef class Binance(MarginExchange):
+    cdef dict _cleanup_positions_dict(self, dict positions)
```
Comment (GuillaumeDSM) [inline Trading/Exchange/Margin/binance.pxd]: Missing newline 😃

File: Trading/Exchange/Margin/binance.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                'public': 'https://testnet.binancefuture.com/fapi/v1',
+                'private': 'https://testnet.binancefuture.com/fapi/v1',
+                'fapiPublic': 'https://testnet.binancefuture.com/fapi/v1',
+                'fapiPrivate': 'https://testnet.binancefuture.com/fapi/v1'
+            }
+
+        self.client.setSandboxMode(is_sandboxed)
+
+    async def get_open_position(self):
+        return self._cleanup_positions_dict(await self.client.fapiPrivateGetPositionRisk())
```
Comment (GuillaumeDSM) [inline Trading/Exchange/Margin/binance.py]: ```self.client.fapiPrivateGetPositionRisk()``` that's a very meaningful name from ccxt 😂

File: Trading/Exchange/Margin/binance.pxd [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from octobot_trading.exchanges.margin.margin_exchange cimport MarginExchange
+
+cdef class Binance(MarginExchange):
+    cdef dict _cleanup_position_dict(self, dict position)
```
Comment (GuillaumeDSM) [inline Trading/Exchange/Margin/binance.pxd]: missing newline

File: Trading/Exchange/Margin/binance.py [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        try:
+            # If exchange has not position id -> global position foreach symbol
+            if ExchangeConstantsOrderColumns.ID.value not in position:
+                position[ExchangeConstantsOrderColumns.ID.value] = position[
+                    ExchangeConstantsOrderColumns.SYMBOL.value]
+            if ExchangeConstantsPositionColumns.QUANTITY.value not in position \
+                    and self.BINANCE_FUTURE_QUANTITY in position:
+                position[ExchangeConstantsPositionColumns.QUANTITY.value] = position[self.BINANCE_FUTURE_QUANTITY]
+            if ExchangeConstantsPositionColumns.TIMESTAMP.value not in position:
+                position[ExchangeConstantsPositionColumns.TIMESTAMP.value] = 0  # TODO
```
Comment (GuillaumeDSM) [inline Trading/Exchange/Margin/binance.py]: binance is not sending the timestamp value ?

File: Trading/Exchange/Margin/bybit.py [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+
+from octobot_trading.enums import ExchangeConstantsOrderColumns, ExchangeConstantsPositionColumns
+from octobot_trading.exchanges.margin.margin_exchange import MarginExchange
+
+
+class Bybit(MarginExchange):
```
Comment (GuillaumeDSM) [inline Trading/Exchange/Margin/bybit.py]: 👍

File: Trading/Exchange/Margin/binance.py (hunk 4 above)
Comment (Herklos) [reply Trading/Exchange/Margin/binance.py]: Good question, I'll take a look at this

**Pull request closed without merging**